regtests/run_spark_sql.sh

```diff
@@ -13,11 +13,19 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+# -----------------------------------------------------------------------------
+# Purpose: Launch the Spark SQL shell to interact with Polaris.
+# -----------------------------------------------------------------------------
 #
+# Usage:
+#   Without arguments: Uses local filesystem.
+#   With two arguments: Uses S3.
```

Comment:
we should probably do a real usage string rather than this,

usage: ./run_spark_sql.sh [s3-path aws-role]

s3-path: Location on S3 to .....
aws-role: arn?role? used for authenticating catalog?client?both?

Reply:
Added more context to explain it is a role used by the catalog